webapp/app/[locale]/_components/header.tsx

```diff
@@ -19,27 +20,30 @@ type Props = {
   toggleMenu: () => void
 }
 
-export const Header = ({ isMenuOpen, toggleMenu }: Props) => (
-  <header
-    className="md:h-17 md:py-4.5 flex h-14 items-center border-b border-solid
+export const Header = function ({ isMenuOpen, toggleMenu }: Props) {
+  const href = useTunnelOperationByConnectedWallet()
+  return (
+    <header
+      className="md:h-17 md:py-4.5 flex h-14 items-center border-b border-solid
      border-neutral-300/55 bg-white px-3 py-3 md:bg-transparent md:px-0"
-  >
-    <div className="h-6 w-6 md:hidden">
-      <Link href="/tunnel">
-        <HemiSymbol />
-      </Link>
-    </div>
-    <div className="hidden pl-6 md:block">
-      <TunnelTabs />
-    </div>
-    <WalletConnection />
-    <button
-      className="flex h-8 w-8 cursor-pointer items-center
-      justify-center rounded-lg border border-neutral-300/55 md:hidden"
-      onClick={toggleMenu}
-      type="button"
     >
-      {isMenuOpen ? <CloseIcon /> : <HamburgerIcon />}
-    </button>
-  </header>
-)
+      <div className="h-6 w-6 md:hidden">
+        <Link href={href}>
+          <HemiSymbol />
+        </Link>
+      </div>
+      <div className="hidden pl-6 md:block">
+        <TunnelTabs />
+      </div>
+      <WalletConnection />
+      <button
+        className="flex h-8 w-8 cursor-pointer items-center
```

Comment:
It could be replaced as
```suggestion
        className="flex size-8 cursor-pointer items-center
```

Reply:
cool, I did not know about it. Thanks for the tip!

Reply:
Sugestion applied here [924d183](https://github.com/hemilabs/ui-monorepo/pull/707/commits/924d183ec85369d3da88209fc6cca7b9c9a75b35)